src/hotspot/share/compiler/compiler_globals.hpp

```diff
@@ -302,6 +302,9 @@
           range(0, 100)                                                     \
                                                                             \
   /* compiler directives */                                                 \
+  product(ccstrlist, AOTCreateOnMethodEntry, "",                            \
```

Comment:
Would "AOTEndTrainingRunOnMethodEntry" be more appropriate? Does "Create" refer to creating the AOT cache? I wonder if we should pick one "action" throughout (create AOT cache or trigger end of training run) and keep it consistent.

Reply:
AOTEndTrainingOnMethodEntry